### DISCUSSION TITLE
Beacon Stations softlock the sub

### DISCUSSION THREAD

**Discussion by Rehexr** [FakeFishGames/Barotrauma#13628] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

This has happened multiple times now. Seems like activiating a beacon station with the sub on top or near it will completely get it stuck and unable to move by any means, even if it should. Can only be fixed by using teleportsub to move it elsewhere, away from the beacon. This has happened regardless of which beacon variant it was. 
(Please mind that the sub is fully operational here, yet it still cant move)

https://user-images.githubusercontent.com/104232152/200140718-30354dab-5140-4e71-961a-e757ebb0d32a.mp4

### Reproduction steps

1.Pick a Beacon Mission
2.Park the sub, sitting on top of the beacon
3.Activate the beacon
4.Attempt to move the sub afterwards

### Bug prevalence

Happens regularly

### Version

Faction test branch

### Which operating system did you encounter this bug on?

Windows

**Comment by itchyOwl** [comment]
Possibly related: #4374?

**Comment by Regalis11** [comment]
I'm afraid I couldn't find a way to reproduce this.

Looking at the video, it seems the sonar transducer tower is inside the submarine, which would explain why it can't move. 
![image](https://user-images.githubusercontent.com/14351070/207073447-6a5b5336-6b75-49dc-97d3-015b93e5ddbb.png)


So I think the question is, how did the tower end up inside the submarine? Giving this a quick test, I couldn't find a way to get it to happen - the submarine collides with the tower, so I couldn't get them to overlap. Did you by any chance teleport the submarine on top of the beacon station?

**Comment by Rehexr** [comment]
Huh, certainly didnt teleport th,e sub on top of the beacon, though my only guess, and the only thing I could possibly blame, might be the fact that the sub sat on top is using custon hulls to make a moonpool of sorts, which might mess some things up when the beacon is turned on? Ill give it a try for a moment, and see if thats the issue.

**Comment by Rehexr** [comment]
Yeah, ive had no luck replicating this in the most recent build, even in what id assume to be the identical situation as here, tried it between 3 subs and beacon stations and no luck, suppose it might just not be an issue anymore for some reason?

**Comment by Regalis11** [comment]
No known steps to repro, closing

**Comment by zsh-is-life** [comment]
Issue still exists on version 1.0.20.1. No mods in use. Singleplayer vanilla.
Typhon 2 ended up being stuck to the beacon station's transducer. It some how ended up being wedged into where the bottom right gun is (it was the chaingun in my case). Tried emptying/filling ballasts and tried positive and negative thrust. Submarine was fully powered and operational, except the fact that it could not move at all.
